module/handlebar-helpers.mjs

```diff
@@ -9,6 +9,10 @@ export default function registerHandlebarHelpers() {
 
   // General Handlebars
 
+  Handlebars.registerHelper( 'ed-getTalentSort', ( value ) => {
```

Comment:
Do we really need a helper for that? Or is that something that could be done in `getData`? If possible, put it there, trying as little as possible in the handlebars themselves